_opensearch/segment-replication/configuration

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Segment Replication Configuration
+nav_order: 122
+parent: Segment Replication Feature
+grand_parent: Opensearch
+---
+
+## Segment Replication Configuration
+
+### Enabling the feature flag
+
+#### Enable on a node using a tarball install
+
+The flag is toggled using a new jvm parameter that is set in either `OPENSEARCH_JAVA_OPTS` or in config/jvm.options.
+
+1. Option 1. Update config/jvm.options with:
+
+    ````json
+    -Dopensearch.experimental.feature.replication_type.enabled=true
+    ````
+
+1. Option 2: Use the `OPENSEARCH_JAVA_OPTS` environment variable:
+
+    ````json
+    export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.replication_type.enabled=true"
+    ````
+
+#### Enable with Docker containers
+
+For Docker composer, add the following line to docker-compose.yml underneath the `opensearch-node` and `environment` section:
+
+````json
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.replication_type.enabled=true" # Enables segment replication
+````
+
+#### Setting replication strategy on the index
+
+When creating an index via the REST interface, add `'{"settings" : { "index" : { "replication.type" : "SEGMENT" }}}'` to your payload:
```

Comment:
I think we should call out that you can only set replication strategy to segment replication before you first create an index. Something like:
To set the replication strategy to segment replication, create an index with `replication.type` set to `SEGMENT`:

```json
PUT /my-index1
{
  "settings": {
    "index": {
      "replication.type": "SEGMENT" 
    }
  }
}
```